client-app/ui-kit/components/atoms/checkbox/vc-checkbox.vue

```diff
@@ -97,6 +99,13 @@ function change() {
     emit("change", newArray);
   }
 }
+
+function clickHandler(event: MouseEvent) {
```

Comment:
yes, for click events the `MouseEvent` type is used

Reply:
@click reacts on "space" button also when the input is active.

Reply:
you are right, I googled it and there is a special behavior https://www.w3.org/TR/WCAG20-TECHS/SCR35 when space or enter keypress on focused element fires click event. Anyway, just in case I replaced MouseEvent with more generic Event, because I don't use any properties which specific to mouse events.